pkg/util/validation/errors_test.go

```diff
@@ -28,7 +28,12 @@ func TestNewMetadataUnitTooLongError(t *testing.T) {
 
 func TestNewMaxQueryLengthError(t *testing.T) {
 	err := NewMaxQueryLengthError(time.Hour, time.Minute)
-	assert.Equal(t, "the query time range exceeds the limit (query length: 1h0m0s, limit: 1m0s) (err-mimir-max-query-length). To adjust the related per-tenant limit, configure -store.max-query-length, or contact your service administrator.", err.Error())
+	assert.Equal(t, "the (sharded) query time range exceeds the limit (query length: 1h0m0s, limit: 1m0s) (err-mimir-max-query-length). To adjust the related per-tenant limit, configure -store.max-query-length, or contact your service administrator.", err.Error())
```

Comment:
It's not correct saying "sharded" here. The query may my only split (by time), sharded (by series) or not split/sharded at all. I would just keep the previous "the query time ..."

Reply:
removed it, I'll try to make the runbook better instead, error message is probably not the place to explain everything anyway